### DISCUSSION TITLE
OSError: [WinError 126]

### DISCUSSION THREAD

**Discussion by windkwbs** [open-webui/open-webui#4668] [General]
OSError: [WinError 126] The specified module was not found. Error loading "D:\open-webui-0.3.13\installer_files\conda\envs\Lib\site-packages\torch\lib\fbgemm.dll" or one of its dependencies.

**Comment by glsup** [comment]
Have you ever got it pass this error? I'm seeing the same.
I see someone closed an issue with "Use a VirtEnv", which is too cryptic for me...
https://github.com/open-webui/open-webui/discussions/4516

**Comment by bsaid1** [comment]
If you want to run on the CPU and tried many solutions but it is not working, you can try to use this command:

`pip install torch==2.2.2 torchvision==0.17.2 torchaudio==2.2.2 --index-url https://download.pytorch.org/whl/cpu`
It will to download torch with CPU only support.

**Reply by windkwbs** [reply]
GUP
